ferretdb/ferretdb.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ferretdb
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/FerretDB/FerretDB/internal/clientconn"
+	"github.com/FerretDB/FerretDB/internal/handlers/registry"
+	"github.com/FerretDB/FerretDB/internal/util/logging"
+)
+
+// Config contains a backend connection strings.
+type Config struct {
+	Handler     string
+	PostgresURL string
```

Comment:
A tiny inconsistency: in some places, PostgreSQL is referred to as `Postgres`, in others as `pg`. I see it was referred as `Pg` before (e.g. `opts.PgPool`), so let's use `Pg` everywhere?

Reply:
> `Postgres`
sorry I can't find this used as a connection string.

> `pg`

There are also no such connection sting.

> `opts.PgPool`

`opts.PgPool` has a `postgresURL` as a connection string.

would you please give exact references of inconsistency?

Reply:
I believe we should rather call it `Postgres` or `Pg` but not both. On this particular line, you call it `Postgres`, on others you call it `Pg`. It's not connection string specific, it's about everything. 

Like, why do we call it `Pg` everywhere, but for connection string we decided to call it `Postgres`? 

This is not super important though, so if you disagree, and calling it both `Postgres` and `Pg` is ok, feel free to close the thread.

Reply:
> I believe we should rather call it `Postgres` or `Pg` but not both. On this particular line, you call it `Postgres`, on others you call it `Pg`.

sorry I can't find pgURL anywhere... What did you mean? 

> Like, why do we call it `Pg` everywhere, but for connection string we decided to call it `Postgres`?

Again, could you please provide me with references or let's have a call? 

> This is not super important though, so if you disagree, and calling it both `Postgres` and `Pg` is ok, feel free to close the thread.